content: Create FW Epilogue mission for Barmy Edwards (#5478)
endless-sky/endless-sky: data/human/free worlds epilogue.txt

			`	After you talk for a while longer, Danforth wishes you luck and sends you on your way. "I'll be in touch if we ever need your assistance in the future," he says.`
				decline

mission "FW Epilogue: Barmy"
	minor
	landing
	name `Weapon Testing`
	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
	source "Rust"
	to offer
		has "free worlds plot completed"
	on offer
		conversation
			`As you complete your landing procedure you recall having come here to assist in some testing for new weapons, back in the thick of the war. Do you want to check in on Barmy Edwards, the lead engineer?`
			choice
				`	(Yes.)`
				`	(No.)`
					defer
			scene "scene/loc0"
			`	You give your name to the receptionist, and she directs you to the hangar where the weapons group does their design and prototyping. Barmy Edward shakes your hand warmly and says, "Ah, <last>. It's been too long! How fortunate for me, I'm actually just putting the finishing touches on a new prototype. You wouldn't be too busy to assist me in some practical testing would you?`

> 			`	"I'm not sure about this, what would you have me do?" you say.`

Author: Converting to a choice with each choice leading to the same text. How's this sound, `	"Being a target for experimental weapons? Of course I will! What do you need?"`?

> 			`	"I'm not sure about this, what would you have me do?" you say.`

Reviewer: Looks good!

			`	"Not much out of the ordinary from our tests in the past," he says. "You'll find a ship of ours in orbit that goes by 'Hunk', it has been fitted with a few of these new weapons. Your job will be to fight it and disable it, then tell us what you think of its combat capabilities and transfer your ships logs so we can analyze the fight."`
			choice
				`	"Alright, I suppose for old times sake I can help out."`
				`	"Sorry, I just wanted to check in and see how you were doing."`
					goto catchup

> 			`	He turns to you and says with a smirk, "Now mind you, we just want you to disable the ship, not destroy it. We've recommissioned it from retirement, so it's not top of the line but all the same I want to be able to keep using it for future tests. Understand?"`

Reviewer: ```suggestion
			`	He turns to you and smirks. "Now mind you, we just want you to disable the ship, not destroy it. We've recommissioned it from retirement, so it's not top of the line, but I want to use it for future tests. Understand?"`
```
OR
```suggestion
			`	He turns to you and says with a smirk, "Now mind you, we just want you to disable the ship, not destroy it. We've recommissioned it from retirement, so it's not top of the line, but, all the same, I want to be able to keep using it for future tests. Understand?"`
```
Top is some style suggestions; bottom is purely grammar corrections. Up to you.

> 			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`

Reviewer: I think this is a spot that could use a response from the player. At the very least, could we add `you nod` or something so it's understood that `<first> <last>` acknowledges it?

				accept
			label catchup

> 			`	"I see, well I'm sure we can find someone else up to the task. Regardless captain <last>, I assure you this new weapon vastly improves on the outdated Plasma Cannon, this will put Kraz Cybernetics in the big leagues!"`

Reviewer: ```suggestion
			`	"I see. Well, I'm sure we can find someone else up to the task. Regardless Captain <last>, I assure you this new weapon vastly improves on the outdated Plasma Cannon. This will put Kraz Cybernetics in the big leagues!"`
```

> 			`	As his tangent about his new invention continues, you think to yourself about the old times and the war, and all the people you've met along the way.`

Reviewer: ```suggestion
			`	As his tangent about his new invention continues, you think to yourself about the old times, the war, and all the people you've met along the way.`
```

> 			`	"How about we get out of here <first>? My work here is done today, I know a homely little bar not far from here."`

Reviewer: ```suggestion
			`	"How about we get out of here, <first>? My work is done for today, and I know a homely little bar that's only a short walk."`
```
Triple `here`.

			choice

> 				`	"Of course, it's not like there's a war to worry about anymore."`

Reviewer: ```suggestion
				`	"Of course. It's not like there's a war to worry about anymore."`
```

> 			`"Ha well of course, the war kept all of us plenty busy." He leads you out of the facility, all the while telling an anecdote that must be humor for an engineer.`

Reviewer: ```suggestion
			`"Ha! Well of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
```

> 			`	You follow Barmy into this bar he was telling you of, even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks, Barmys order catches you off guard and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle and you both tell stories of your early lives, specifically before leaving the ground. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`

Reviewer: ```suggestion
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve outdated equipment.`
```
`specifically before leaving the ground` and `tells how he had some exciting adventures in his youth among the stars` feels contradictory unless you're going to specify the conversation topic shifts, in which case I think some more writing would be necessary.

> 			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there, I was lucky to get out myself"`

Reviewer: ```suggestion
			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
```

> 			`	"No way you're from New Boston too! What are the chances?" you hastily respond.`

Reviewer: ```suggestion
			`	"No way you're from New Boston too! What are the chances?" you respond.`
```
`hastily` implies some sort of rushed/frantic sense, which doesn't seem appropriate for the conversation.

> 			`	He chuckles, "Small galaxy I suppose." You both have a great time hearing of each others early lives but you wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and you head back to your ship.`

Reviewer: ```suggestion
			`	He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
```

				decline
		event "plasma repeater advanced" 95
		event "plasma repeater basics" 155
	npc disable save
		government "Test Dummy"
		personality disables heroic staying
		ship "Leviathan (Plasma Repeater)" `Hunk`
	on complete

> 		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon, increased fire rate even at lower damage per hit comes out with better overall results.`

Reviewer: ```suggestion
		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
```

		conversation
			scene "scene/plasma_scene"

> 			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun much like an old Plasma Cannon, but somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs alot hotter, you see," he says. "But the results are impressive."`

Reviewer: ```suggestion
			`You head to the hangar to meet up with Barmy Edward and find him tinkering with a large gun. It resembles an old Plasma Cannon, but it is somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs much lot hotter, you see," he explains. "But the results are impressive."`
```

			choice
				`	"What was that?"`
					goto what
				`	"It looks impressive, but it didn't do too much damage."`
					goto damage
			label what

> 			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That, boy, was the Plasma Repeater. Of course, it wasn't easy to find where opportunity lies when everyone could have taken a crack at it."`

Reviewer: ```suggestion
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to find where opportunity lies when everyone could have taken a crack at it."`
```
`boy` specifies gender. Big no-no. Also, what does the last sentence mean? Don't quite understand it.

Author: The last sentence was a bit shaky, I admit. Rewording it to make sense. How does `Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."` sound?

Reviewer: `countless fools` sounds a bit harsh, but it could also just be Barmy's character. Regardless, sounds good!

			`	"When will it be available for sale?" you ask.`
				goto available
			label damage

> 			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma we can offload many more bursts onto a target, the effects on said target result in faster build up of heat and of course damage. Of course, big ships can ward of these effects much longer than a smaller ship can, it's always been a notable issue with the orginal design, but it is a hell of a lot more impressive."`

Reviewer: ```suggestion
			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Of course, big ships can ward off these effects much longer than a smaller ship can. It's always been a notable issue with the original design, but it's a hell of a lot more impressive."`
```

> 			`	"That is an interesting idea," you say. "Are they available for sale yet?"`

Reviewer: ```suggestion
			`	"That is an interesting idea," you reply. "Are they available for sale yet?"`
```

			label available
			`	"Well, I should have an initial model for sale in a month or so," he says. "I'll be interested to see what uses you or anyone else might find for them. Anyway, thanks for your help."`
			`	"How about we celebrate with some drinks? I know a quaint little bar not far from here."
			choice

> 				`	"I have time, lets celebrate!"`

Reviewer: ```suggestion
				`	"I have time; lets celebrate!"`
```

> 				`	"I'll pass on that but it was good checking in on you."`

Reviewer: ```suggestion
				`	"I'll pass on that, but it was good checking in on you."`
```

> 					decline

Reviewer: Does this need to be a `goto end` or something? This isn't in the `on offer` section, this is `on complete`. Could be me just not knowing the syntax, though.

Reviewer 2: you need to use one of the conversation enders (accept, decline, defer) if you want to end a conversation without it reaching the end

			`	You follow Barmy into this bar he was telling you of, even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks, Barmys order catches you off guard and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle and you both tell stories of your early lives, specifically before leaving the ground. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there, I was lucky to get out myself"`
			`	"No way you're from New Boston too! What are the chances?" you hastily respond.`

> 			`	He chuckles, "Small galaxy I suppose." You both have a great time hearing of each others early lives but you wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and you head back to your ship.`

Reviewer: Same suggestion above would apply here.

			`	"How about we get out of here <first>? My work is done for today, an I know a homely little bar that's only a short walk."`
			choice
				`	"Of course. It's not like there's a war to worry about anymore."`
			`"Ha! Well of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`

> 			`	"No way you're from New Boston too! What are the chances?" you respond.`

Reviewer 2: This should probably also be a choice. After all, there's no guarantee that the player would respond with this much surprise.

			`	He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
				decline
		event "plasma repeater advanced" 95
		event "plasma repeater basics" 155
	npc disable save
		government "Test Dummy"
		personality disables heroic staying
		ship "Leviathan (Plasma Repeater)" `Hunk`
	on complete
		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
		conversation
			scene "scene/plasma_scene"

> 			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with a large gun. It resembles an old Plasma Cannon, but it is somewhat different. Presumably, this is the weapon that you were just attacked with in orbit. "It runs much hotter, you see," he explains. "But the results are impressive."`

Reviewer 2: ```suggestion
			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with the modified plasma cannon you saw when you first entered. "It runs much hotter, you see," he explains. "But the results are impressive."`
```

			choice

> 				`	"What was that?"`

Reviewer 2: ```suggestion
				`	"Was that a plasma cannon? Or something else?"`
```

					goto what

> 				`	"It looks impressive, but it didn't do too much damage."`

Reviewer 2: ```suggestion
				`	"It's not winning any damage records, that's for sure."`
```

					goto damage
			label what

> 			`	"Heh, it's been a while since we've seen a good upgrade to the cannon." he says. "That was the Plasma Repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`

Reviewer 2: ```suggestion
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon," he says. "That was the plasma repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
```

			`	"When will it be available for sale?" you ask.`
				goto available
			label damage

> 			`	"Well, you are famous for a reason I suppose." he says. "The idea is that by shrinking the concentrations of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Of course, big ships can ward off these effects much longer than a smaller ship can. It's always been a notable issue with the original design, but it's a hell of a lot more impressive."`

Reviewer 2: ```suggestion
			`	"Well, you are one of the captains that drove off the Pug." he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Of course, big ships can ward off these effects much longer than a smaller ship can, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`
```

> 			`	"That is an interesting idea," you reply. "Are they available for sale yet?"`

Reviewer 2: ~stop stealing lines from flamethrower testing~
```suggestion
			`	"That's a unique direction," you reply. "Are they available for sale yet?"`
```

			label available

> 			`	"Well, I should have an initial model for sale in a month or so," he says. "I'll be interested to see what uses you or anyone else might find for them. Anyway, thanks for your help."`

Reviewer 2: ```suggestion
			`	"Thanks to you, it should be in stock in a few months," he replies. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`
```

> 			`	"How about we celebrate with some drinks? I know a quaint little bar not far from here."

Reviewer 2: ```suggestion
```

			choice
				`	"I have time; lets celebrate!"`
				`	"I'll pass on that, but it was good checking in on you."`
					decline

> 			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he doesn't stick out from the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`

Reviewer 2: same advice for the next few lines as the advice for this section in the on offer

	minor
	landing
	name `Weapon Testing`
	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
	source "Rust"
	to offer
		has "free worlds plot completed"
	on offer
		conversation
			`As you complete your landing procedure, you recall having come here to assist in some testing for new weapons, back in the thick of the war. Do you want to check in on Barmy Edwards, the lead engineer?`
			choice
				`	(Yes.)`
				`	(No.)`
					defer
			scene "scene/loc0"
			`	You give your name to the receptionist, and she directs you to a hangar where you find Barmy Edward, who is working on a weapon that looks like a modified plasma cannon. When he notices you, he gets up from his work and shakes your hand warmly. "Ah, <last>. It's been too long! How fortunate for me! I'm actually just putting the finishing touches on a new prototype. You wouldn't be too busy to assist me in some practical testing, would you?`
			choice
				`	"I'm not sure about this. What would you have me do?"`
				`	"Being a target for experimental weapons? Just tell me what you need!"`

> 			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight and disable it, tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`

Reviewer 3: ```suggestion
			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by <npc>. It's been fitted with a few of these new weapons. Your job will be to fight and disable it, tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`
```

> 			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight it and disable it, then tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`

Reviewer: ```suggestion
			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight and disable it, tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`
```
This last sentence is kind of tough to work out grammatically, but I think this takes out all the bumps.

			choice
				`	"Alright, I suppose for old times' sake I can help out."`
				`	"Sorry, I just wanted to check in and see how you were doing."`
					goto catchup
			`	He turns to you and smirks. "We've recommissioned the ship from retirement, so it's not top of the line, but try to keep it in one piece for future use. Understand?"`
			choice
				`	"Of course! I'll take care of it."`
			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`
				accept
			label catchup
			`	"I see. Well, I'm sure we can find someone else up to the task. Regardless, Captain <last>, I assure you this new weapon vastly improves on the outdated plasma cannon. This'll put Kraz Cybernetics in the big leagues!"`

> 			`	As his ramblings about his new invention continue, you think to yourself about the old times, the war, and all the people you've met along the way.`

Reviewer 3: This line doesn't really seem like it fits here. I'd just toss it, maybe reuse it somewhere else if you can.
```suggestion
```

			`	"How about we get out of here, <first>? My work is done for today, and I know a homely little bar that's only a short walk."`
			choice
				`	"Of course. It's not like there's a war to worry about anymore."`

> 			`"Ha! Well, of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`

Reviewer 3: ```suggestion
			`	"Ha! Well, of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
```

> 			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`

Reviewer 3: ```suggestion
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars, but found his passion tinkering and finding ways to improve on outdated equipment.`
```

			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
			choice
				`	"It wasn't perfect, but at least it wasn't quite so noisy down on the ground."`

> 				`	"Nothing like it is now. I've got the whole universe ahead of me!"`

Reviewer 3: Nothing to mention about Edward also being from New Boston?

Author: I created a new option for the player that addresses Barmy being from New Boston, wondering if you have anything to say on the options that existed before, or if it still feels empty.

			`	"You wouldn't think to have met someone from your very own birth planet, eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
				decline

> 		event "plasma repeater advanced" 95
> 		event "plasma repeater basics" 155

Reviewer 3: I believe these should be in an on decline.
```suggestion
	on decline
		event "plasma repeater advanced" 95
		event "plasma repeater basics" 155
```

Author: But since the `on complete` section also has `decline` endpoints, would that not run it twice or necessitate using the same event timer for both branches?

Reviewer 3: `on decline` only triggers when the mission itself is declined, not when any conversation related to the mission hits a decline endpoint.

	npc save disable
		government "Test Dummy"
		personality disables heroic staying

> 		ship "Leviathan (Plasma Repeater)" `Hunk`

Reviewer 3: ```suggestion
		ship "Leviathan (Plasma Repeater)" "Hunk"
```

	on complete
		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
		conversation
			scene "scene/plasma scene"
			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with the modified plasma cannon you saw when you first entered. "It runs much hotter, you see," he explains. "But the results are impressive."`
			choice
				`	"Was that a plasma cannon? Or something else?"`
					goto what
				`	"It's not winning any damage records, that's for sure."`
					goto damage
			label what
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon," he says. "That was the plasma repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
			`	"When will it be available for sale?" you ask.`
				goto available
			label damage

> 			`	"Well, you are one of the revered captains that drove off the Pug" he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Big ships will usually ward off these effects much longer than a smaller ship would, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`

Reviewer: ```suggestion
			`	"Well, you are one of the revered captains that drove off the Pug," he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Big ships will usually ward off these effects much longer than a smaller ship would, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`
```

			`	"That's a unique direction," you reply. "Are they available for sale yet?"`
			label available
			`	"Thanks to you, it should be in stock in a few months," he answers. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`
			choice
				`	"I have time; lets celebrate!"`
				`	"I'll pass on that, but it was good checking in on you."`
					decline
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`

> 			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`

Reviewer: ```suggestion
			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
```

			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars, but found his passion tinkering and finding ways to improve on outdated equipment.`
			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
			choice
				`	"Really? I didn't think I knew anybody that grew up there."`
					goto youtoo
				`	"It wasn't perfect there, but at least it wasn't quite so noisy down on the ground."`
				`	"Nothing like it is now. I've got the whole universe ahead of me!"`
			`	"You wouldn't think to have met someone from your very own birth planet, eh?" He chuckles. "Small galaxy, I suppose."`

> 			label youtoo

Reviewer 3: This looks reversed. This first choice should lead into "You wouldn't think" while the other two choices should lead to the end.

Author: I thought it would be weird to have "I didn't think I knew anybody that grew up there" followed by "You wouldn't think to have met someone from your very own birth planet, eh?"

Felt to me that both addresses it in some small part at least but the players choice that addresses it covers it enough.

> 			label youtoo

Reviewer 3: I'd just use `label end` to keep label names one word.

			`	As his ramblings about his new invention continue, you think to yourself about the old times, the war, and all the people you've met along the way.`
			`	"How about we get out of here, <first>? My work is done for today, and I know a homely little bar that's only a short walk."`
			choice
				`	"Of course. It's not like there's a war to worry about anymore."`
			`"Ha! Well, of course, the war kept all of us plenty busy." He leads you out of the facility while telling an anecdote that must be humor for an engineer.`
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`

> 			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`

Reviewer: ```suggestion
			`	He's shocked to discover your origins on New Boston. "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
```

			choice
				`	"It wasn't perfect, but at least it wasn't quite so noisy down on the ground."`

> 				`	"Nothing like it is now. I've got the whole universe ahead of me now!"`

Reviewer: ```suggestion
				`	"Nothing like it is now. I've got the whole universe ahead of me!"`
```
Double `now`

> 			`	"You wouldn't think to have met someone from your very own birth planet eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`

Reviewer: ```suggestion
			`	"You wouldn't think to have met someone from your very own birth planet, eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
```

				decline
		event "plasma repeater advanced" 95
		event "plasma repeater basics" 155
	npc save disable
		government "Test Dummy"
		personality disables heroic staying
		ship "Leviathan (Plasma Repeater)" `Hunk`
	on complete
		log `Assisted Barmy Edward with another weapon test: this time, an impressive upgrade to the plasma cannon. Increased fire rate, even at lower damage per hit, comes out with better overall results.`
		conversation
			scene "scene/plasma scene"
			`You head to the hangar to meet up with Barmy Edward, and find him tinkering with the modified plasma cannon you saw when you first entered. "It runs much hotter, you see," he explains. "But the results are impressive."`
			choice
				`	"Was that a plasma cannon? Or something else?"`
					goto what
				`	"It's not winning any damage records, that's for sure."`
					goto damage
			label what
			`	"Heh, it's been a while since we've seen a good upgrade to the cannon," he says. "That was the plasma repeater. Of course, it wasn't easy to make a better iteration of something that countless fools have failed to improve on."`
			`	"When will it be available for sale?" you ask.`
				goto available
			label damage

> 			`	"Well, you are one of the revered captains that drove off the Pug" he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Big ships usually, will ward off these effects much longer than a smaller ship can, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`

Reviewer: ```suggestion
			`	"Well, you are one of the revered captains that drove off the Pug," he says. "The idea is that by shrinking the bolts of superheated plasma, we can offload many more bursts onto a target. The effects on said target result in faster buildup of heat and, of course, damage. Big ships will usually ward off these effects much longer than a smaller ship would, but that's always been a notable issue with the original design. Besides, the repeater is a hell of a lot more impressive."`
```

			`	"That's a unique direction," you reply. "Are they available for sale yet?"`
			label available

> 			`	"Thanks to you, it should be in stock in a few months," he replies. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`

Reviewer: ```suggestion
			`	"Thanks to you, it should be in stock in a few months," he answers. "I'm sure you're past the point where our weapons are useful, but then again, not everybody has a jump drive." He grins. "How about we celebrate with some drinks? I know a quaint little bar not far from here."`
```
The immediate proceeding line (if you go to `label damage` first and follow from there) has `reply` in it as well, so this'll make it a little less repetitive.

			choice
				`	"I have time; lets celebrate!"`
				`	"I'll pass on that, but it was good checking in on you."`
					decline
			`	You follow Barmy into the bar. Even with the smear of grease still on his face, he fits in with the crowd. You both stand at the bar and order your drinks. Barmy's order catches you off guard, and he notices your expression. "What? A man who works with his hands can't enjoy a drink with some age?" You chuckle, and you both tell stories of your early lives. He tells how he had some exciting adventures in his youth among the stars but found his passion tinkering and finding ways to improve on outdated equipment.`
			`	He's shocked to discover your origins on New Boston, "You're really from that backwater planet? I'm surprised you were able to escape the racket they have going on there. I was lucky to get out myself!"`
			choice
				`	"It wasn't perfect, but at least it wasn't quite so noisy down on the ground."`

> 				`	"Nothing like it is now. I've got the whole universe ahead of me now!"`

Reviewer: ```suggestion
				`	"Nothing like it is now. I've got the whole universe ahead of me!"`
```

> 			`	"You wouldn't think to have met someone from your very own birth planet eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`

Reviewer: ```suggestion
			`	"You wouldn't think to have met someone from your very own birth planet, eh?" He chuckles. "Small galaxy, I suppose." You both have a great time hearing of each other's early lives, but you eventually wrap up your visit after a long night of bouncing stories back and forth. You say your goodbyes and head back to your ship.`
```
